config/crd/bases/tower.ansible.com_ansibleinventories.yaml

```diff
@@ -33,21 +33,45 @@ spec:
             properties:
               name:
                 type: string
+                description: The name to use for the inventory
               description:
                 type: string
+                description: The description for the inventory
               organization:
                 type: string
+                description: Organization name, ID, or named URL the inventory belongs to
+              instance_groups:
+                type: array
```

Comment:
Good catch, this should have been an array from the start since an inventory can be associated with multiple InstanceGroups.  Thank you for correcting the sample Inventory yaml as well.